src/acceptance/Makefile

```
@@ -5,7 +5,7 @@ MAKEFLAGS := -s
 GO := GO111MODULE=on GO15VENDOREXPERIMENT=1 go
 GO_NOMOD := GO111MODULE=off go
 
-GO_VERSION = $(shell go version | sed --expression='s/^[^0-9.]*\([0-9.]*\).*/\1/')
+GO_VERSION = $(shell go version | sed -e s/^[^0-9.]*\([0-9.]*\).*/\1/')
```

This does not work because the closing `'` got deleted 👀 

```suggestion
GO_VERSION = $(shell go version | sed -e 's/^[^0-9.]*\([0-9.]*\).*/\1/')
```

By the way, what's wrong with the more verbose param-name (`--expression`)? :D